recordedit/recordEdit.app.js

```diff
@@ -263,14 +263,12 @@
                             throw response;
                         });
                     } else if (session) {
-                        var notAuthorizedMessage = "You are not authorized to Update entities.";
                         var notAuthorizedError = new Error(notAuthorizedMessage);
                         // user logged in but not allowed (forbidden)
                         notAuthorizedError.code = errorNames.forbidden;
 
                         throw notAuthorizedError;
                     } else {
-                        var notAuthorizedMessage = "You are not authorized to Update entities.";
```

Comment:
Since you removed the variable above, this case will fail/error out with no message defined.

Reply:
I have added the error message code back for this case

Reply:
Maybe you misunderstood what @RFSH had said. The errors in `ermrestJS` are ways of wrapping JS errors, but you have to still provide the status and message that will be displayed. See [`errors.js`](https://github.com/informatics-isi-edu/ermrestjs/blob/master/js/errors.js) or the [`.doc`](https://github.com/informatics-isi-edu/ermrestjs/blob/master/doc/api.md#ERMrest.TimedOutError).

You need to be cautious though. In the case of Unauthorized (401), we use an unauthorized message. In the case of Forbidden (403), we still use an unauthorized message and status. We don't want to tell the user they are "forbidden" instead we use "unauthorized" to notify them in both cases.

We still want to use the proper error type (Unauthorized or Forbidden) but want the message to be the same in both cases.